Mathlib/NumberTheory/ModularForms/LevelOne.lean

```diff
@@ -0,0 +1,72 @@
+/-
+Copyright (c) 2024 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+import Mathlib.NumberTheory.Modular
+import Mathlib.NumberTheory.ModularForms.SlashInvariantForms
+import Mathlib.NumberTheory.ModularForms.CongruenceSubgroups
+
+/-!
+# Level one modular forms
+
+This file contains results specific to modular forms of level one, ie. modular forms for `SL(2, ℤ)`.
+
+TODO: Add finite dimisionality of these spaces of modular forms.
+
+-/
+
+open UpperHalfPlane ModularGroup SlashInvariantForm ModularForm CongruenceSubgroup Complex
+local notation "SL(" n ", " R ")" => Matrix.SpecialLinearGroup (Fin n) R
+local notation  "Γ(" n ")"  => CongruenceSubgroup.Gamma n
+
+/--The map from `SL(2, ℤ)` to `Γ(1)`. Its preferable to work with this, since levels of modular
```

Comment:
```suggestion
/-- The map from `SL(2, ℤ)` to `Γ(1)`. Its preferable to work with this, since levels of modular
```
what is the alternative to "working with this"? As an outside reader, this comment is not completely clear to me.